bodhi/models/models.py

```diff
@@ -1649,6 +1649,27 @@ def check_requirements(self, session, settings):
 
         return True, "All checks pass."
 
+    @property
+    def has_negative_karma(self):
+        """Check for negative karma, Returns True if the update has negative karma"""
+        for comment in self.comments:
+            if not comment.anonymous:
+                if comment.karma < 0:
+                    return True
+        return False
+
+    def diasble_autopush_for_critical_update(self, db):
```

Comment:
Does anything call this method? I don't see any calls to it in this change, and it's a new method.